tests/cli/dorq/test_repos.py

```diff
@@ -260,21 +261,82 @@ def test_list_wf_run_ids(monkeypatch):
 
 
 class TestConfigRepo:
+    class TestUnitConfigRepo:
```

Comment:
The enclosing class already provides a namespace with `ConfigRepo` in it.
```suggestion
    class TestUnit:
```